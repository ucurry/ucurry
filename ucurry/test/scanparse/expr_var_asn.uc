-- def -> expr -> variable_name = expr
x = 2;
a = 3;
