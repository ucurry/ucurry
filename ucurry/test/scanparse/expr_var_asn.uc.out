Printing Roundtrip
x = 2;
a = 3;
